packages/terra-date-picker/src/react-datepicker/year_dropdown_options.jsx

```diff
@@ -48,11 +51,11 @@ export default class YearDropdownOptions extends React.Component {
   renderOptions = () => {
     var selectedYear = this.props.year
     var options = this.state.yearsList.map(year =>
-      <div className="react-datepicker__year-option"
+      <div className={cx('react-datepicker-year-option')}
         key={year}
         ref={year}
         onClick={this.onChange.bind(this, year)}>
-        {selectedYear === year ? <span className="react-datepicker__year-option--selected">✓</span> : ''}
+        {selectedYear === year ? <span className="react-datepicker-year-option--selected">✓</span> : ''}
```

Comment:
This isn't getting compiled

Reply:
fixed here https://github.com/cerner/terra-framework/pull/794/commits/4054daa7ca8c140580b44e284194fb98e2a2e659.